test/battle/ai_check_viability.c

```diff
@@ -179,7 +179,6 @@ AI_SINGLE_BATTLE_TEST("AI chooses moves with secondary effect that have a 100% c
     PARAMETRIZE { ability = ABILITY_SERENE_GRACE; }
 
     GIVEN {
-        AI_LOG;
```

Comment:
Is this an accident?

Reply:
no, this is something I (At least I believe it was me) added accidentally some time ago so I removed it.